frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -622,6 +642,35 @@ const getFilters = (t: TFunction): RowFilter<NodeRowItem>[] => [
       return input.selected?.some((r) => nodeRoles.includes(r));
     },
   },
+  {
+    filterGroupName: t('console-app~CPU Architecture'),
+    type: 'cpu-arch',
+    reducer: (obj) => (isCSRResource(obj) ? 's390x' : getNodeCpuArch(obj)),
+    items: [
+      { id: 'amd64', title: t('console-app~amd64') },
+      { id: 'ppc64le', title: t('console-app~ppc64le') },
+      { id: 'arm64', title: t('console-app~arm64') },
+      { id: 's390x', title: t('console-app~s390x') },
+    ],
```

Comment:
probably should be amd64 not s390x on 648

I think we should generate this list dynamically based only include the ones that are available.